experiments/AMIP/modular/components/land/bucket_utils.jl

```diff
@@ -152,3 +152,32 @@ get_field(sim::BucketSimulation, ::Val{:beta}) =
 get_field(sim::BucketSimulation, ::Val{:albedo}) =
     ClimaLSM.surface_albedo(sim.model, sim.integrator.u, sim.integrator.p)
 get_field(sim::BucketSimulation, ::Val{:area_fraction}) = sim.area_fraction
+
+
+# The surface air density is computed using the atmospheric state at the first level and making ideal gas
+# and hydrostatic balance assumptions. The land model does not compute the surface air density so this is
+# a reasonable stand-in.
+function update_field!(sim::BucketSimulation, ::Val{:air_density}, field)
+    parent(sim.integrator.p.bucket.ρ_sfc) .= parent(field)
+end
+
+function update_field!(sim::BucketSimulation, ::Val{:turbulent_energy_flux}, field)
+    parent(sim.integrator.p.bucket.turbulent_energy_flux) .= parent(field)
+end
+function update_field!(sim::BucketSimulation, ::Val{:turbulent_moisture_flux}, field)
+    ρ_liq = (LSMP.ρ_cloud_liq(sim.model.parameters.earth_param_set))
+    parent(sim.integrator.p.bucket.evaporation) .= parent(field ./ ρ_liq)
+end
```

Comment:
Why do we divide by `\rho_liq` here, when all of the other `update_field!` methods just set the array to `field`?

Reply:
This was a particular specification of the land model. The `coupler_field` should contain what we decide to support in the coupler (now the default is what comes out of the `SurfaceFluxes.jl`). And any model-specific unit conversions etc can be dealt with in the `init_` files.